config/manager/manager.yaml

```diff
@@ -30,6 +30,7 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      hostNetwork: true
```

Comment:
Is `hostNetwork=true` due to peerpod-ctrl and is this really needed ?

Reply:
yes, AFAIR for it's needed in order to use Instance Metadata Service